### ISSUE TITLE
Builds fail for multi-project builds if environment property file is not in rootDir

### ISSUE THREAD

**Issue by scottdotau** [stevesaliman/gradle-properties-plugin#14]
Hey Guys,

This is a question more than anything, was just wondering if this is expected behaviour. I am currently building a 'whitelabel' project, and am using this plugin (thanks!) for filtering my properties files. 

Only thing that has been annoying me a little, is the need to have `gradle-<environmentName>.properties` in the rootDir project (though is fine if it doesn't exist in the subproject - inherits gradle.properties). 

Build fails if this is not there, and as I am setting `environmentName` dynamically, the rootDir gets quite messy (`gradle-<environmentName>.properties` files not just in sub-projects, but rootDir also - mainly empty).

Example:

```
+ shared
+ project1
|- gradle.properties
|- gradle-env1.properties
|- gradle-env3.properties
+ project2
|- gradle.properties
|- gradle-env1.properties
|- gradle-env2.properties
|- gradle-env3.properties
gradle.properties
gradle-env1.properties
gradle-env2.properties
```

I would expect all of these to work (even if environment properties file is missing - as the `gradle.properties` file exists), however:
`gradle :project1:build -PenvironmentName=env1` works
`gradle :project1:build -PenvironmentName=env2` works
`gradle :project1:build -PenvironmentName=env3` fails (No environment files were found for the 'env3' environment)

Basically, I don't want to have to have all of the environment property files within my rootDir. However, if you pass through environmentName - it MUST be there (but not in subprojects - even if they are the target you are building), or it fails.
